docs/examples/customization/custom-errors/custom-default-backend.yaml

```diff
@@ -36,7 +36,7 @@ spec:
     spec:
       containers:
       - name: nginx-error-server
-        image: quay.io/kubernetes-ingress-controller/custom-error-pages-amd64:0.3
+        image: gcr.io/k8s-staging-ingress-nginx/nginx-errors:0.48.1
```

Comment:
One last change: 

The image path should be `k8s.gcr.io/ingress-nginx/nginx-errors:0.48.1` 

Anyway, I'm ok to fix this later as we promote the image :)